document/src/main/resources/config/liquibase/changelog/202221021-add-search-field-table.xml

```diff
@@ -0,0 +1,16 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
Seems like we have some duplicate work:
https://github.com/valtimo-platform/valtimo-backend-libraries/pull/355/files#diff-4f74a6f262610c5200d54201b733ae0e34f02717756242a46f506ff607adc8a3